config/config.go

```diff
@@ -49,6 +49,7 @@ type (
 			Kernel      string
 			Environ     []string
 			Volumes     []string
+			Labels      []string `envconfig:"DRONE_AGENT_LABELS"`
```

Comment:
I've been working on something similar, used 
```go
Labels      map[string]string `envconfig:"DRONE_AGENT_LABELS"`
```

instead, `envconfig` will handle `DRONE_AGENT_LABELS=mem:high,cpu:load`.

Reply:
this is true.  we did not use maps for environment variables (not sure why) but we did use maps for other configuration parameters. I think in this case using a map makes sense.

Reply:
nice, i didn't realize `envconfig` handled maps. i'll get these all updated